scss/detail.scss

```diff
@@ -44,6 +58,12 @@
   width: 200px;
 }
 
+.book-detail-container {
+  display: flex;
+  justify-content: center;
+  height: 768px;
```

Comment:
I am not sure about .book-detail-container has a fixed height, why did you do it?

Reply:
That was unnecesary. I'll remove it :+1: